apps/filebrowser/src/filebrowser/views.py

```diff
@@ -203,6 +203,7 @@ def download(request, path):
     setattr(response, 'redirect_override', True)
   else:
     response = StreamingHttpResponse(file_reader(fh), content_type=content_type)
+    response["Cache-Control"] = 'must-revalidate' # Browsers must check with hue for updates
```

Comment:
Curious: 
If I understand correctly, `must-validate` will only validate with the server for stale or old cache. But that means it will be reused if cache is fresh. If we are not setting some cache expiry age, then will this take any default value for the cache and that can cause the same problem? Maybe any browser specific default value is present? Can this scenario be present?

Also if our end-goal is anyway to ask the browser to ask Hue server for new file, then can we try with `no-cache` or `no-store` to mitigate the above scenario? @amitsrivastava

Reply:
Your question on cache expiry age is valid. Using just `must-revalidate` without specifying an age has worked fine for a customer that was facing the trouble. But, I think we should `max-age=0` to not rely on browser defaults at all.

`no-cache` vs `must-revalidate`: My understanding that `must-revalidate` will continue to use the cached version as long as the modified-timestamp of the file is same as that of cached version. While the `no-cache` will completely disable caching. So, I feel that `must-revalidate` will be better for performance reasons. Do you think otherwise?

Reply:
But if we are going to add `max-age=0` with `must-revalidate`, then it means the same thing? In both ways, the browser will ask the Hue server for new file every-time. I feel this should not have any performance delta.

From https://developer.mozilla.org/en-US/docs/Web/HTTP/Caching#force_revalidation:

>max-age=0 means that the response is immediately stale, and must-revalidate means that it must not be reused without revalidation once it is stale — so, in combination, the semantics seem to be the same as no-cache.
>
>However, that usage of max-age=0 is a remnant of the fact that many implementations prior to HTTP/1.1 were unable to handle the no-cache directive — and so to deal with that limitation, max-age=0 was used as a workaround.
>
>But now that HTTP/1.1-conformant servers are widely deployed, there's no reason to ever use that max-age=0 and must-revalidate combination — you should instead just use no-cache.

Reply:
Awesome. So, I will change it to `no-cache`. Thanks.

Reply:
@amitsrivastava @Harshg999 
Latest Hue is using Django 3.2.14, and it will use HTTP 1.1
https://github.com/django/django/blob/3.2.14/django/core/servers/basehttp.py#L86
But do we need to fix older version? Here is https://github.com/cloudera/hue/blob/master/desktop/core/ext-py/Django-1.11.29/django/core/servers/basehttp.py#L84-L85

Reply:
@wing2fly I checked logs of Hue running in CDH 6.x and it is showing `HTTP/1.1` there.

Reply:
Thanks @Harshg999 for confirming